code/style.css

```diff
@@ -1,4 +1,194 @@
+            margin-left: 20px;
+            font-size: 20px;
+            color: #1761a0
+        }
+        /*--------------main--------------------*/
+        
+        .wrapper {
+            position: relative;
+            min-height: 100%;
+        }
+        
+        .info {
+            width: 100%;
+            height: auto;
+            display: flex;
+        }
+        
+        .info img {
+            max-height: 500px;
+            width: 70%;
+            /* align-self: center; */
+        }
+        
+        .info .text {
+            align-self: center;
+            margin-left: auto;
+            margin-right: auto;
+            text-align: center;
+            width: 30%;
+        }
+        
+        .btn {
+            background: #f0a418;
+            padding: 10px;
+            color: #fff;
+            margin-left: auto;
+            border-radius: 15px;
+            color: #1761a0;
+        }
+        
+        .destinations {
+            width: 100%;
+            height: auto;
+            margin-top: 40px;
+            display: flex;
+            flex-wrap: wrap;
+            justify-content: center;
+        }
+        
+        .destination-box {
+            width: 20%;
+            height: auto;
+            background: #ecf2f7;
+            margin: 20px;
+            padding: 10px;
+            box-sizing: border-box;
+            box-shadow: 0 10px 10px #999;
+            transition: box-shadow 0.3s ease
+        }
+        
+        .destination-box:hover {
+            box-shadow: 5px 10px 10px #999;
+        }
+        
+        .destination-box img {
+            height: 200px;
+            width: 100%;
+        }
+        
+        .destinations h2 {
+            text-align: center;
+        }
+        
+        .staycation {
+            height: auto;
+            display: grid;
+            grid-template-columns: repeat(6, 1fr);
+            margin-top: 100px;
+            margin-bottom: 100px;
+            background: #ecf2f7;
+            grid-gap: 5px;
+        }
+        
+        .staycation .grid2 {
+            padding: 50px;
+            text-align: center;
+        }
+        
+        .staycation img {
+            width: 100%;
+            height: 200px;
+        }
+        /* ------------footer----------------- */
+        
+        footer {
+            background: #c7d9e8;
+            height: 80px;
+            text-align: center;
+            padding-top: 30px;
+            color: #1761a0;
+            width: 100%;
+            justify-content: center;
+            margin-top: 100px;
+            position: absolute;
+            bottom: 0;
+            left: 0;
+        }
+        
+        footer span {
+            color: #f0a418;
+        }
+        /*---------------- Media queries----------------- */
+        /* media query for tablets */
+        
+        @media screen and (max-width: 1024px) {
+            .destination-box {
+                width: 40%;
+            }
```

Comment:
With this media query value, your destination box will always be 40% of the screen width when the screen is less than 1025px. This makes the boxes look a bit slim in for mobile users (40% of 500px is not that much). Did you style your project mobile first? By doing that the media queries become a bit easier to grasp in the beginning. I know that they are a mouthful 😅